src/v2/ion/i18nTransformer.js

```diff
@@ -135,7 +152,11 @@ const updateLabelForUiSchema = (remediation, uiSchema) => {
     ? `.${remediation.relatesTo.value.type}`
     : '';
   const i18nPrefix = `${remediation.name}${authenticatorType}.`;
-  const i18nPath = `${i18nPrefix}${uiSchema.name}`;
+  let i18nPath = `${i18nPrefix}${uiSchema.name}`;
+
+  if (uiSchema.type === 'text' && uiSchema.name.indexOf('questionKey') >= 0) {
```

Comment:
just double check, will this only override predefined SQ but not custom SQ?

Reply:
yeah added a logic to weed it out.